api/agent_client.go

```diff
@@ -12,6 +12,9 @@ import (
 	"time"
 )
 
+// This is a special keyword supported on backend for polling from the current default queue in the cluster.
+const DefaultQueueKey = "_default"
```

Comment:
It probably doesn't need to be an exported name, especially if it is an internal protocol detail.

```suggestion
const defaultQueueKey = "_default"
```